src/main/java/edu/greenblitz/robotName/commands/shooter/ShootByShootingPower.java

```diff
@@ -0,0 +1,33 @@
+package edu.greenblitz.robotName.commands.shooter;
+
+
+import edu.greenblitz.robotName.commands.shooter.flyWheel.FlyWheelCommand;
+import edu.greenblitz.robotName.commands.shooter.flyWheel.StopFlyWheel;
+import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.FlyWheelConstants;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class ShootByShootingPower extends FlyWheelCommand {
+
+    public ShootByShootingPower() {
+    }
+
+    @Override
+    public void execute() {
+        flyWheel.setPower(FlyWheelConstants.SHOOTING_POWER);
+    }
+
+    @Override
+    public boolean isFinished() {
+        return flyWheel.isAtVelocity(FlyWheelConstants.SHOOTING_POWER);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        new SequentialCommandGroup(
```

Comment:
Try to avoid scheduling commands inside end functions of basic commands. You can define a command for the shooting part and use the SequentialCommandGroup to create a bigger command.